backend/app/processing/data_loader.py

```
@@ -161,6 +161,26 @@ def load_excel(file_path: str) -> List[str]:
+    sentences = re.split(r'(?<=[.!?])\s+', text)
+    chunks = []
+    for sentence in sentences:
+        if not sentence:
+            continue
+        if chunks and len(chunks[-1]) + len(sentence) < max_length:
+            chunks[-1] += ' ' + sentence
+        else:
+            chunks.append(sentence)
+    return chunks
+
+def process_page_text(index: int, text: str) -> List[str]:
```

Ruff (0.8.2)

Use `list` instead of `List` for type annotation

Replace with `list`

(UP006)